### PR TITLE
Build GNOME and Plasma 5 netboot images as well

### PR DESCRIPTION
**Pull request by Enzime**
Depends on https://github.com/NixOS/nixpkgs/pull/150363 :)

### CODE REVIEW

File: build-images.sh [hunk 1]
```diff
@@ -33,6 +41,8 @@ main() {
   readarray -t assets < <(
     build_kexec_bundle "$tag" "$arch" "$tmp"
     build_netboot_image "$tag" "$arch" "$tmp"
+    build_gnome_netboot_image "$tag" "$arch" "$tmp"
+    build_plasma5_netboot_image "$tag" "$arch" "$tmp"
```
Comment (Mic92) [inline build-images.sh:45]: We need to check if those images can be still uploaded to github / fit in RAM.

Comment (Enzime) [reply build-images.sh:45]: They're about the same size as the corresponding ISOs (~2GiB for GNOME, 1.4GiB for Plasma 5). I believe that should still be fine for both GitHub and RAM :smile:

Comment (Mic92) [reply build-images.sh:45]: Ok. I am wondering why our normal kexec image is than so big (1.3GB)...

Comment (Enzime) [reply build-images.sh:45]: Looks like something changed between `20.09` and `21.05` where `20.09` is only 600MiB and `21.05` is 1.3GiB for the `kexec-bundle`

**Comment by Mic92** [comment]
what's the status on this?

**Comment by Mic92** [comment]
Closing this, since it seems stale. Would be still happy to merge it, if you or someone else picks this up.

**Pull request closed without merging**